ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -467,14 +473,22 @@
         - lldpd
         - lldp-syncd
 
+    - name: Remove iptables rule to drop BGP SYN Packet from Peer
+      shell: "iptables -D INPUT -j DROP -p tcp --destination-port bgp"
+      become: true
+
+    - name: Remove ip6tables rule to drop BGP SYN Packet from Peer
+      shell: "ip6tables -D INPUT -j DROP -p tcp --destination-port bgp"
+      become: true
+
     - name: Enable bgpd
       become: yes
-      lineinfile: dest=/etc/quagga/daemons
-                  regexp=^bgpd=.*$
-                  line='bgpd=yes'
+      supervisorctl: state=started name={{item}}
+      delegate_to: "{{ ansible_host }}_bgp"
+      with_items:
+        - bgpd
```

Comment:
specify 'bgpd' directly in the 'supervisorctl' command

Reply:
@neethajohn Updated.